android/app/src/androidTest/java/io/parity/signer/EthkeyBridgeTest.java

```diff
@@ -0,0 +1,84 @@
+package io.parity.signer;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+import com.facebook.react.bridge.Promise;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContext;
+import com.facebook.react.bridge.WritableMap;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.test.core.app.ApplicationProvider;
+
+class TestPromise implements Promise {
+    private Object expectedValue;
+    private boolean shouldBeRejected;
+
+    public TestPromise(@Nullable Object expectedValue, boolean shouldBeRejected) {
+        this.expectedValue = expectedValue;
+        this.shouldBeRejected = shouldBeRejected;
+    }
+
+    public void resolve(@Nullable Object value) {
+        assertEquals(value, this.expectedValue);
+    }
+
+    public void reject(String code, String message) {
+        assertTrue(this.shouldBeRejected);
+    }
+
+    public void reject(String code, Throwable throwable) {
```

Comment:
Where do we use `reject` method? for simplification we could also do 
```
public void reject(Object... x) {
  assertTrue(this.shouldBeRejected)
}
```

Reply:
Not sure I understand.. we don't use anything in this file, since I didn't find a way to successfully mock a ReactApplicationContext in the test environment.  I included the file in case anyone else wants to make another attempt, but it can be discarded.

Reply:
I mean there are many duplcated function name just because the arguments are different, but arguments are not used, by using `Object... x`, we will ignore them and only have one `reject` function